### ISSUE TITLE
[openstack-cloud-controller-manager] Add an additional-vip subnet-id parameter for LoadBalancer

### ISSUE THREAD

**Issue by maxcoolmakarov** [kubernetes/cloud-provider-openstack#2735]
**Is this a BUG REPORT or FEATURE REQUEST?**:

kind feature

**What happened**:

My cloud provider has it's load balancer setup so the only way to create LB is calling

`openstack loadbalancer create --vip-subnet-id IPv6 --additional-vip subnet-id=IPv4 --flavor amph-failover-standard --name lb1`

Currently your plugin supports setting only --vip-subnet-id parameter throw config file, but I also need to set up --additional-vip subnet-id as well. Could you please add this feature? (I guess the LB address shoud be then obtained form additional-vip subnet, because the vip-subnet-id will give you only IPv6 address.

**Comment by zetaab** [comment]
so https://github.com/kubernetes/cloud-provider-openstack/blob/master/pkg/openstack/loadbalancer.go#L66 https://github.com/kubernetes/cloud-provider-openstack/blob/master/pkg/openstack/loadbalancer.go#L73 https://github.com/kubernetes/cloud-provider-openstack/blob/master/pkg/openstack/loadbalancer.go#L75 any of these does not work?

**Comment by maxcoolmakarov** [comment]
> so https://github.com/kubernetes/cloud-provider-openstack/blob/master/pkg/openstack/loadbalancer.go#L66 https://github.com/kubernetes/cloud-provider-openstack/blob/master/pkg/openstack/loadbalancer.go#L73 https://github.com/kubernetes/cloud-provider-openstack/blob/master/pkg/openstack/loadbalancer.go#L75 any of these does not work?

No, all of this id's are about "primary" network of LB, which in my case is IPv6. And IPv4 is set inside AdditionalVIP
I forked this repo to implement AdditionalVIP support https://github.com/kubernetes/cloud-provider-openstack/compare/master...maxcoolmakarov:cloud-provider-openstack:master
